csp_billing_adapter/utils.py

```diff
@@ -13,29 +13,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+"""utils.py is part of csp-billing-adapter and provides utility functions"""
 import datetime
 
 from dateutil.relativedelta import relativedelta
 
 
 def get_now():
+    """get current time"""
     return datetime.datetime.now(datetime.timezone.utc)
 
 
 def date_to_string(date):
-    return date.isoformat()
+    """convert date to string"""
+    try:
+        return date.isoformat()
+    except (AttributeError, ValueError) as exc:
```

Comment:
Probably should be logging something useful here given that we are catching and then re-raising the exceptions ;-)

Reply:
So my point really is that if we are catching and re-raising the same exception, but not doing anything with it, there is not much point in doing the exception catch/re-raise... Usually in such a case I would be logging that the exception occurred and any additional useful information that is available in the content where the exception was caught...

Reply:
i will add logging to these. any objections?